doc/policies.md

```diff
@@ -40,24 +40,24 @@ policy B. When APIcast receives an HTTP request, it will check the policy chain
 described to see what it should run on each phase:
 - rewrite: execute the function policy A provides for this phase.
 - access: execute the function policy B provides for this phase.
-- content: do nothing. Neither policy A nor B describe what to do.
-- balancer: do nothing. Neither policy A nor B describe what to do.
+- content: do nothing. Neither policy A nor B describes what to do.
```

Comment:
```suggestion
- content: Does nothing. Neither policy A nor B describes what to do.
```

I'm not sure if you want to indicate that the user shouldn't do anything, or the phase does nothing. Can you please reword this and the next line?

Reply:
It refers to the phase as per the context.

Reply:
To facilitate reading, rephrase the last sentence and add a table to describe what APIcast does. This is an example:

```
When APIcast receives an HTTP request, it will check the policy chain
perform the tasks outlined in the following table:

| Phase    | APIcast tasks                                           |
|----------|---------------------------------------------------------|
| rewrite  | Executes the function policy A provides for this phase. |
| access   | Executes the function policy B provides for this phase. |
| content  | None. Neither policy A nor B  describes what to do.     |
| balancer | None. Neither policy A nor B  describes what to do.     |
```

Reply:
The tabular representation does make it look better and more understandable. Made the changes. Please check.